### PR TITLE
Add some new weekend flavour text

### PR DESCRIPTION
**Pull request by SarenCurrie**
Topical...

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -93,6 +93,12 @@
     }, {
       "activity" : "helping a friend move in! I knew these muscles would come in handy someday! :3",
       "outcome" : "Whew, that was tough. They sure do have a lot of stuff! Hey, that rhymed too! :o"
+    }, {
+      "activity" : "going to stock up on dried food and toilet paper! A friend of mine has a cough and I'm worried I might get quarantined. -_-",
+      "outcome" : "They were out of everything. I came home with an exipred roast chicken and two grapes."
```
Comment (franomai) [inline activities.json]: Supreme kek

File: activities.json [hunk 2]
```diff
@@ -93,6 +93,12 @@
     }, {
       "activity" : "helping a friend move in! I knew these muscles would come in handy someday! :3",
       "outcome" : "Whew, that was tough. They sure do have a lot of stuff! Hey, that rhymed too! :o"
+    }, {
+      "activity" : "going to stock up on dried food and toilet paper! A friend of mine has a cough and I'm worried I might get quarantined. -_-",
+      "outcome" : "They were out of everything. I came home with an exipred roast chicken and two grapes."
+    }, {
+      "activity" : "being told not to leave the house! Well I've got a lot of anime to watch at least. :P",
```
Comment (franomai) [inline activities.json]: Change to "Well, at least I have a lot of anime to watch. :P" and I'll merge

**Comment by SarenCurrie** [comment]
You better squash when you merge @franomai

**Review by franomai — APPROVED** [review]
Looks goodo

**Pull request merged**